cfgov/v1/documents.py

```diff
@@ -25,7 +23,7 @@ class FilterablePagesDocument(Document):
         'name': fields.TextField(),
         'slug': fields.KeywordField()
     })
-    title = fields.TextField(attr='title', analyzer=ngram_tokenizer)
+    title = fields.TextField(attr='title')
```

Comment:
This analyzer was causing more harm then good, a more refined solution is coming with the expanded search shortly anyways. Just removing this for now.